contracts/script/Utils.s.sol

```diff
@@ -62,6 +62,36 @@ contract UtilsScript is Script {
         return (filePath, obj3);
     }
 
+    function generateDeploymentOutput(
+        string memory contractName,
+        uint256 contractSalt,
+        address contractAddress,
+        address multisig,
+        string memory approvalProcessId,
+        string memory viaType
+    ) external returns (string memory filePath, string memory data) {
+        string memory outputDir = string.concat(
+            vm.projectRoot(),
+            "/contracts/script/output/defenderDeployments/",
```

Comment:
Should we allow this to be a `.env` variables instead of being hard coded here to make it more configurable?

Reply:
Thanks Sneh. Currently, There's an open issue to deal with defender deployment logs https://github.com/EspressoSystems/espresso-sequencer/issues/1286 but currently when we deploy with openzeppelin defender, the deployment logs aren't written locally (whereas foundry normally does this). So this is a workaround and is not expected to be used more dynamically. I've put that string into a variable since it has been referenced multiple times in the script. https://github.com/EspressoSystems/espresso-sequencer/pull/1718/commits/c80c26ed1147291e29fd86f3ae2818d1eab1304b